plugins/services/src/js/validators/MarathonAppValidators.js

```diff
@@ -137,6 +137,23 @@ const MarathonAppValidators = {
 
     // No errors
     return [];
+  },
+
+  /**
+   * @param {Object} app - The data to validate
+   * @returns {Array} Returns an array with validation errors
+   */
+  mustNotContainUris(app) {
+    if (ValidatorUtil.isDefined(app.uris) &&
+        ValidatorUtil.isDefined(app.fetch)) {
```

Comment:
Yes. Because:

* If only `uris` is present, that's a a legit old way of pasting stuff (still accepted by mesos) so it must be accepted
* If only `fetch` is present, that's the correct way of doing things
* Both `fetch` and `uris` will appear ONLY if you are editing a service and you try to add `uris`. This time, marathon will not accept a JSON with both fields present. Therefore I am prompting the user to remove the `uris` field in favor of `fetch`